pyvista/plotting/renderer.py

```diff
@@ -505,15 +578,55 @@ def add_axes(self, interactive=None, line_width=2,
         interacitve : bool
             Enable this orientation widget to be moved by the user.
 
-        line_width : int
+        line_width : int, optional
             The width of the marker lines
 
-        box : bool
+        box : bool, optional
             Show a box orientation marker. Use ``box_args`` to adjust.
             See :any:`pyvista.create_axes_orientation_box` for details.
 
-        opacity : int or float, optional
-            The opacity of the marker.
+        xcolor : str, optional
```

Comment:
These look unfixed (x_color vs xcolor), maybe the commits aren't pushed yet.  But I think the description should be something like "Color of x axis label", which I missed before

Reply:
Fixed in https://github.com/pyvista/pyvista/pull/1479/commits/6a74d3cf1f3a5c230a154ec78b4178f713d33588